.gitignore

```diff
@@ -56,7 +56,8 @@ vignettes/*.pdf
 *.pdf
 *.jpeg
 .my.cnf
+.tex
```

Comment:
What are .tex files?

Reply:
Latex file. I use it to write documents. They should not be on github for ownership issues.